packages/console/src/containers/ConsoleContent/hooks.ts

```diff
@@ -0,0 +1,62 @@
+import { Prompt, useLogto } from '@logto/react';
+import { getTenantOrganizationId } from '@logto/schemas';
+import { useContext, useEffect, useState } from 'react';
+
+import { TenantsContext } from '@/contexts/TenantsProvider';
+import useCurrentTenantScopes from '@/hooks/use-current-tenant-scopes';
+import useRedirectUri from '@/hooks/use-redirect-uri';
+import { saveRedirect } from '@/utils/storage';
+
+const useTenantScopeListener = () => {
+  const { currentTenantId } = useContext(TenantsContext);
+  const { clearAccessToken, clearAllTokens, getOrganizationTokenClaims, signIn } = useLogto();
+  const [tokenClaims, setTokenClaims] = useState<string[]>([]);
+  const [isDecodingClaims, setIsDecodingClaims] = useState(false);
```

Comment:
this can be merged as:

```suggestion
  const [tokenClaims, setTokenClaims] = useState<string[]>();
```

The use `undefined` for the loading (unset) state

Reply:
Ah, good idea!